File: js/log/log.go
package log

import (
	log "github.com/Sirupsen/logrus"
)

func Log(logger *log.Logger, t, msg string, fields map[string]interface***REMOVED******REMOVED***) ***REMOVED***
	e := logger.WithFields(log.Fields(fields))
	switch t ***REMOVED***
	case "error":
		e.Error(msg)
	case "warn":
		e.Warn(msg)
	case "info":
		e.Info(msg)
	case "debug":
		e.Debug(msg)
	***REMOVED***
***REMOVED***

File: js/lib/api/log.js
"use strict";

$log = $log || ***REMOVED******REMOVED***;

$log.debug = function() ***REMOVED***
	return $log.log.apply(this, _.concat(['debug'], arguments));
***REMOVED***

$log.info = function() ***REMOVED***
	return $log.log.apply(this, _.concat(['info'], arguments));
***REMOVED***

$log.warn = function() ***REMOVED***
	return $log.log.apply(this, _.concat(['warn'], arguments));
***REMOVED***

$log.warning = function() ***REMOVED***
	return $log.warn.apply(this, arguments);
***REMOVED***

$log.error = function() ***REMOVED***
	return $log.log.apply(this, _.concat(['error'], arguments));
***REMOVED***
